src/services/user-group/services/courseGroupModelService.js

```diff
@@ -0,0 +1,61 @@
+const service = require('feathers-mongoose');
+const {
+	iff, isProvider, disallow,
+} = require('feathers-hooks-common');
+const auth = require('@feathersjs/authentication');
+
+const { enableQuery, enableQueryAfter } = require('../../../hooks');
+const {	courseGroupModel } = require('../model');
+
+const courseGroupModelService = service({
+	Model: courseGroupModel,
+	paginate: {
+		default: 25,
+		max: 100,
+	},
+	lean: { virtuals: true },
+});
+
+const print = (context) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5387400)